README.md

```diff
@@ -264,6 +264,125 @@ safe_create_partitioned_table :table, type: :range, key: :example_column, infer_
 end
 ```
 
+#### safe\_partman\_create\_parent
+
+Safely configure a partitioned table to be managed by [pg\_partman](https://github.com/pgpartman/pg_partman).
+
+This method calls the [create\_parent](https://github.com/pgpartman/pg_partman/blob/master/doc/pg_partman.md#creation-functions) partman function with some reasonable defaults and a subset of user-defined overrides.
+
+The first (and only) positional arg maps to `p_parent_table` in the `create_parent` function.
+
+The rest are keyword args with the following mappings:
+
+- `key` -> `p_control`. Required: `true`
```

Comment:
I used `key` here to be consistent with the method signature of `safe_create_partitioned_table`. However, we haven't released a new version since that method was added so we _could_ change it in both places without breaking backwards compatibility if you feel strongly

Reply:
I think that would be valuable. I did later notice that method also, and it's slightly less meaningful there, but I think it will improve readability in both cases (potentially a good amount).